osrf/vrx-docker
# Change sensor references to component references

Related to osrf/vrx#373 Changed sensor references to component references where appropriate, This includes references in `prepare_team_wamv.bash`.

Modified example teams' `component_config.yaml` files as well to include the ball shooter and to demonstrate different component layouts

#### To Test
In `vrx-docker`, run `./prepare_team_wamv.bash example_team` which should create the WAM-V URDF.

To test the generated WAM-V, run `roslaunch vrx_gazebo sydney_regatta.launch urdf:=/path/to/ws/vrx-docker/generated/team_generated/example_team/example_team.urdf`
